financial-templates-lib/logger/SpyTransport.js

```diff
@@ -0,0 +1,26 @@
+// This transport enables unit tests to validate values passed to the winston using a Sinon Spy.
```

Comment:
This is the custom winston transport containing the spy. By simply calling `this.spy(info)` the spy will capture what is emitted from winston which can be used to validate messages from the unit tests.